src/cli/onefuzz/debug.py

```diff
@@ -28,6 +28,7 @@
 
 EMPTY_SHA256 = "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
 ZERO_SHA256 = "0" * len(EMPTY_SHA256)
+DAY_TIMESPAN = "PT24H"
```

Comment:
Are there docs we can link to for this?

Checking my understanding: this ends up in the generated KQL, so in general needs to be an ISO 8601 time interval. The default value is only really specifies a duration on its own, but it gets interpreted as a time interval in context as "the last <duration> since now()".

Reply:
It's included in the help for the debug.logs.query.  I'll add it for these new commands.